helm-charts/common/retriever-usvc/README.md

```diff
@@ -46,13 +46,13 @@ curl http://localhost:7000/v1/retrieval  \
 
 ## Values
 
-| Key                    | Type   | Default                | Description |
-| ---------------------- | ------ | ---------------------- | ----------- |
-| image.repository       | string | `"opea/retriever-tgi"` |             |
-| service.port           | string | `"7000"`               |             |
-| REDIS_URL              | string | `""`                   |             |
-| TEI_EMBEDDING_ENDPOINT | string | `""`                   |             |
-| global.monitoring      | bool   | `false`                |             |
+| Key                      | Type   | Default                  | Description       |
+| ------------------------ | ------ | ------------------------ | ----------------- |
+| image.repository         | string | `"opea/retriever-tgi"`   |                   |
+| service.port             | string | `"7000"`                 |                   |
+| RETRIEVER_COMPONENT_NAME | string | `"OPEA_RETRIEVER_REDIS"` | vector DB backend |
```

Comment:
RETRIEVER_BACKEND:
* "REDIS"
* "MILVUS"

could be more user friendly.

Reply:
done